webpack.config.client.js

```diff
@@ -1,6 +1,13 @@
 /* eslint-disable global-require */
 const AssetsPlugin = require('assets-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const dotenv = require('dotenv');
```

Comment:
Should this not be in the base `webpack.config.js` file? That way it's made available whichever of client/server we're compiling.